modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/support/Support.kt

```diff
@@ -172,10 +188,10 @@ class Support @Inject constructor(
         withContext(Dispatchers.IO) {
             intent.type = "text/html"
             intent.putExtra(Intent.EXTRA_EMAIL, arrayOf("support@pocketcasts.com"))
-            val isPlus = subscriptionManager.getCachedStatus() is SubscriptionStatus.Plus
+            val isPaid = subscriptionManager.getCachedStatus() is SubscriptionStatus.Paid
             intent.putExtra(
                 Intent.EXTRA_SUBJECT,
-                "$subject v${settings.getVersion()} ${if (isPlus) " - Plus Account" else ""}"
+                "$subject v${settings.getVersion()} ${if (isPaid) " - Plus Account" else ""}"
```

Comment:
I think we need to use the account-type logic you created on line 90 here as well.